README.md

```diff
@@ -509,6 +534,43 @@ for the `regal lint` command:
 
 **Note:** all CLI flags override configuration provided in file.
 
+## Project Roots
+
+While many projects consider the project's root directory (in editors often referred to as **workspace**) their
+"main" directory for policies, some projects may contain code from other languages, policy "subprojects", or multiple
+[bundles](https://www.openpolicyagent.org/docs/latest/management-bundles/). While most of Regal's features works
+independently of this — linting, for example, doesn't consider where in a workspace policies are located as long as
+those locations aren't [ignored](#ignoring-files-globally) — some features, like automatically
+[fixing](https://docs.styra.com/regal/fixing) violations, benefit from knowing when a project contains multiple roots.
+
+To provide an example, consider the
+[directory-package-mismatch](https://docs.styra.com/regal/rules/idiomatic/directory-package-mismatch) rule, which states
+that a file declaring a `package` path like `policy.permissions.users` should also be located in a directory structure
+that mirrors that package, i.e. `policy/permissions/users`. When a violation against this rule is reported, the
+`regal fix` command, or its equivalent [Code Action](#regal-language-server) in editors, may when invoked remediate the
+issue by moving the file to the correct location. But where should the `policy/permissions/users` directory *itself*
+reside?
+
+By default, the answer to that question would be the **project**, or **workspace** root. But if the file was found
+in a subdirectory containing a **bundle**, the directory naturally belongs under that *bundle's root* instead. The
+`roots` configuration option under the top-level `project` object allows you to tell Regal where these roots are,
+and have features that depend on this information work as expected.
+
+```yaml
+project:
+  roots:
+    - bundle1
+    - bundle2
+```
+
+The configuration file is not the only way Regal may determine project roots. Other ways include:
+
+- A directory containing a `.manifest` file will automatically be registered as a root
```

Comment:
Changing "By default" to "Normally", as I don't think using .manifest files is a norm.. even though we should help encourage that.

Honestly, I've not made up my mind yet on whether explicit config for roots or .manifest files is my preferene. I have also been pondering whether explicit configuration should override *anything* else. I think that perhaps it should... as if you explicitly tell us "use bundle1 as root", maybe we shouldn't go and "use bundle1 and bundle3" as roots 🤔 

But I don't feel like this would be terrible to change later, so I'm good with what we got.